content/en/docs/internal-documentation/incident-management.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Incident Management
+linkTitle: Incident Management
+---
+
+An incident refers to an event that can happen at any given time and
+ may cause a decrease in the quality or complete outage of one or
+ more of our services. Internal or external customers, our monitoring
+ and alerting systems, or a member of the SRE team can raise an incident.
+
+Preparedness for major incidents is crucial. We have established the
+ following Incident Management processes to ensure SREs can follow predetermined procedures:
+
+- [Incident Management Process](https://source.redhat.com/groups/public/service-delivery/service_delivery_wiki/incident_management_process)
+
+- [Incident Response Cheatsheet](https://github.com/openshift/ops-sop/blob/master/policies/incident_response.asciidoc)
+
+- [Automated Incident Management Process (WebRCA)](https://source.redhat.com/groups/public/service-delivery/service_delivery_wiki/automated_incident_management_process)
+
+## Coverage
+
+Layered Products SRE (LPSRE) provides 24x7 coverage and support
+with primary and secondary on-call SREs responsible for handling production-related issues.
```

Comment:
"production-related" issues is ambiguous especially with regards to what services we support and what aspects of that service we commit to support 24x7.

I would just stop at `...provides 24x7 coverage and support`.